src/components/ebay-menu/index.js

```diff
@@ -0,0 +1,101 @@
+const assign = require('core-js-pure/features/object/assign');
+const indexOf = require('core-js-pure/features/array/index-of');
+const findIndex = require('core-js-pure/features/array/find-index');
+const scrollKeyPreventer = require('makeup-prevent-scroll-keys');
+const rovingTabindex = require('makeup-roving-tabindex');
+const eventUtils = require('../../common/event-utils');
+const NodeListUtils = require('../../common/nodelist-utils');
+const template = require('./template.marko');
+
+module.exports = require('marko-widgets').defineComponent({
+    template,
+    getInitialState(input) {
+        return assign({}, input, {
+            items: (input.items || []).map(item => assign({}, item))
+        });
+    },
+    onRender({ firstRender }) {
+        this.contentEl = this.getEl('menu');
+
+        if (firstRender) {
+            this.tabindexPosition = 0;
+        }
+
+        if (this.state.type !== 'fake') {
+            this.rovingTabindex = rovingTabindex.createLinear(this.contentEl, 'div', {
+                index: this.tabindexPosition, autoReset: null
+            });
+
+            scrollKeyPreventer.add(this.contentEl);
+        }
+    },
+    onBeforeUpdate() {
+        this._handleDestroy();
+    },
+    onDestroy() {
+        this._handleDestroy();
+    },
+    _handleDestroy() {
+        if (this.state.type !== 'fake') {
+            this.rovingTabindex.destroy();
+            scrollKeyPreventer.remove(this.contentEl);
+        }
+    },
+    toggleItemChecked(itemEl) {
+        const itemIndex = indexOf(itemEl.parentNode.children, itemEl);
+        const item = this.state.items[itemIndex];
+        const currentIndex = this.state.items.findIndex(radioItem => radioItem.checked);
+
+        if (this.state.type === 'radio' && itemIndex !== currentIndex) {
+            this.state.items.forEach((eachItem, i) => {
+                eachItem.checked = i === itemIndex;
```

Comment:
(Minor) I know this is valid, but could you wrap the `i === itemIndex` in parens, to prevent people from getting confused if they're not as smart as you.  :)